.packit.yaml

```diff
@@ -58,8 +58,8 @@ jobs:
 
   - job: tests
     trigger: pull_request
```

Comment:
Can we run this test on demand? 
not on each PR? please take a look here 
https://packit.dev/posts/manual-triggering#manual-only-triggering-of-jobs

Add the following 
```
  skip_build: true
  manual_trigger: true
```

Reply:
done